### PR TITLE
Soutenir

### PR DESCRIPTION
**Pull request by AmelMac**

### CODE REVIEW

File: src/components/Header/Navbar/index.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default function Navbar() {
             <Link to="/action/stageup">Stage-Up</Link>
           </DropdownItems>
         </NavDropdown>
-        |<a href="/soutien">Nous Soutenir</a>|
+        |<a href="/soutenir">Nous Soutenir</a>|
```
Comment (AmjadLille) [inline src/components/Header/Navbar/index.jsx]: replace a by Link

**Review by AmjadLille — CHANGES_REQUESTED** [review]

File: src/components/Header/Navbar/index.jsx (hunk 1 above)
Comment (AmelMac) [reply src/components/Header/Navbar/index.jsx]: ok! :)

**Review by AmjadLille — DISMISSED** [review]

File: src/pages/soutenir/index.jsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  BContainer,
+  Liste,
+  BigLetter,
+  Button,
+  Button1,
+} from "./style";
+
+const Soutenir = () => {
+  return (
+    <>
```
Comment (LoicBrassart) [inline src/pages/soutenir/index.jsx]: Tip: If your component has a single parent container (here, the aptly named `<SoutenirContainer>`), you don't need the `<>` tag ;-)

File: src/pages/soutenir/style.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+  width: 100vw;
+  margin-bottom: 5vh;
+`;
+
+export const Container1 = styled.div`
+width: 32%;
+background-color: white;
+border-radius: 5px;
+margin: 1vw;
+padding: 1vh;
```
Comment (LoicBrassart) [inline src/pages/soutenir/style.js]: You forgot my Prettify ;_;

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
You forgot to use Prettier... Too bad, no Evils beyond that

**Review by AmjadLille — DISMISSED** [review]

File: src/components/Header/Navbar/index.jsx [hunk 4]
```diff
@@ -30,8 +30,8 @@ export default function Navbar() {
             <Link to="/action/stageup">Stage-Up</Link>
           </DropdownItems>
         </NavDropdown>
-        |<Link to="/soutenir">Nous Soutenir</Link>|
-        <a href="/actualite">Notre Actualité</a>|
+        |<Link href="/soutien">Nous Soutenir</Link>|
+        <Link href="/actualite">Notre Actualité</Link>|
```
Comment (LoicBrassart) [inline src/components/Header/Navbar/index.jsx]: Wait, why did you change that ?

File: src/pages/soutenir/style.js (hunk 3 above)
Comment (LoicBrassart) [reply src/pages/soutenir/style.js]: Still no Prettify ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
I think you broke something

**Review by AmjadLille — DISMISSED** [review]

**Review by AmjadLille — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**